src/test/java/guitests/DialogMessageTests.java

```diff
@@ -40,6 +40,25 @@ public void showYesNoWarningDialogTest() throws ExecutionException, InterruptedE
     }
 
     @Test
+    public void showYesNoConfirmationDialog_tryYesAndNo_getCorrectValue() throws ExecutionException, InterruptedException {
+        FutureTask<Boolean> yesTask = new FutureTask<>(() ->
+                DialogMessage.showYesNoConfirmationDialog("Confirm", "Confirm Header", "Confirm Message", "yEs", "nO"));
+        Platform.runLater(yesTask);
+        waitUntilNodeAppears(hasText("Confirm Header"));
+        assertNodeExists(hasText("Confirm Header"));
+        clickOn("yEs");
```

Comment:
I think it was to test that we can get the result that the button was
clicked no matter the content of the button (like more than just Yes or No)
On Apr 8, 2016 9:59 PM, "Damith C. Rajapakse" <notifications@github.com>
wrote:

> In src/test/java/guitests/DialogMessageTests.java
> <https://github.com/HubTurbo/HubTurbo/pull/1485#discussion_r59028253>:
>
> > @@ -40,6 +40,25 @@ public void showYesNoWarningDialogTest() throws ExecutionException, InterruptedE
> >      }
> >
> >      @Test
> > +    public void showYesNoConfirmationDialog_tryYesAndNo_getCorrectValue() throws ExecutionException, InterruptedException {
> > +        FutureTask<Boolean> yesTask = new FutureTask<>(() ->
> > +                DialogMessage.showYesNoConfirmationDialog("Confirm", "Confirm Header", "Confirm Message", "yEs", "nO"));
> > +        Platform.runLater(yesTask);
> > +        waitUntilNodeAppears(hasText("Confirm Header"));
> > +        assertNodeExists(hasText("Confirm Header"));
> > +        clickOn("yEs");
>
> Any idea why this weird capitalization in yEs?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1485/files/7b0a8d2d74a153882115dd07ad60d30bf8d35e43#r59028253>
>